plugins/Utils/windowstatestorage.h

```diff
@@ -44,28 +50,25 @@ class WindowStateStorage: public QObject
     Q_DECLARE_FLAGS(WindowStates, WindowState)
     Q_FLAG(WindowStates)
 
-    WindowStateStorage(const QString& dbName = nullptr, QObject *parent = nullptr);
+    WindowStateStorage(const QString &dbName = nullptr, QObject *parent = nullptr);
```

Comment:
One is not supposed to assign or use `nullptr` with a reference. In this case, use an empty `QString()`. This is actually @dobey's fault, but let's fix it here altogether.

```suggestion
    WindowStateStorage(const QString &dbName = QString(), QObject *parent = nullptr);
```